example/adapters/sequelize.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * This is a very rough-edged example, the idea is to still work with the fact that oidc-provider
+ * has a rather "dynamic" schema. This example uses sequelize with sqlite, and all dynamic data
+ * uses JSON fields. id is set to be the primary key, grantId should be additionaly indexed for
+ * models where these fields are set.
+*/
+
+const Sequelize = require('sequelize'); // eslint-disable-line import/no-unresolved
+
+const sequelize = new Sequelize('databaseName', 'username', 'password', {
+  host: 'databaseHost',
+  dialect: 'postgres',
+});
+
+const models = [
+  'Session',
+  'AccessToken',
+  'AuthorizationCode',
+  'RefreshToken',
+  'DeviceCode',
+  'ClientCredentials',
+  'Client',
+  'InitialAccessToken',
+  'RegistrationAccessToken',
+].reduce((map, name) => {
+  map.set(name, sequelize.define(name, {
+    id: { type: Sequelize.STRING, primaryKey: true },
+    grantId: { type: Sequelize.STRING },
+    userCode: { type: Sequelize.STRING },
```

Comment:
let's only apply this to the DeviceCode model

Reply:
Added that check.